### PR TITLE
Docs: Update visualization topic and template

### PR DESCRIPTION
**Pull request by imatwawana**
Updates Visualization topic under Structure section as well was template

To be merged after [PR 91220](https://github.com/grafana/grafana/pull/91220).

### CODE REVIEW

File: docs/sources/structure/topic-types/visualization/index.md [hunk 1]
```diff
@@ -80,22 +89,17 @@ For visualizations that require a data source, refer to [Visualization topic str
...
-- **Body:** The body may include task/how-to information as needed.
-  The body must include the following sections:
+- **Body** - The body may include task/how-to information as needed. The body must include the following section:
 
-  - **_`<SECTION>`_ options:** For each drop-down section of options in the edit panel, add a section in the topic with the name of the UI section, followed by the word "options".
-    For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options".
+  - **_`<SECTION>`_ options** - For each expandable section of options in the panel edit pane, add a section in the topic with the name of the UI section, followed by the word "options". For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options".
 
-## Visualization naming conventions
+## Naming conventions
```
Comment (imatwawana) [inline docs/sources/structure/topic-types/visualization/index.md:102]:
```suggestion
<!-- vale Grafana.Gerunds = NO -->
## Naming conventions
<!-- vale Grafana.Gerunds = YES -->
```

File: docs/sources/structure/topic-types/visualization/index.md [hunk 2]
```diff
@@ -80,22 +89,19 @@ For visualizations that require a data source, refer to [Visualization topic str
...
-  Don't use longer videos here.
+- **Main visual** - Add a clear screenshot depicting the visualization displaying a typical use case.
 
-- **Body:** The body may include task/how-to information as needed.
-  The body must include the following sections:
+- **Body** - The body may include task/how-to information as needed. The body must include the following section:
 
-  - **_`<SECTION>`_ options:** For each drop-down section of options in the edit panel, add a section in the topic with the name of the UI section, followed by the word "options".
-    For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options".
+  - **_`<SECTION>`_ options** - For each expandable section of options in the panel edit pane, add a section in the topic with the name of the UI section, followed by the word "options". For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options".
```
Comment (imatwawana) [inline docs/sources/structure/topic-types/visualization/index.md:101]:
```suggestion

<!-- vale Grafana.Gerunds = NO -->
```

**Review by jdbaldry — APPROVED** [review]
Nice, thanks for updating this. Would you like to be CODEOWNER on the file so you can review any future changes?

**Comment by imatwawana** [comment]
> Nice, thanks for updating this. Would you like to be CODEOWNER on the file so you can review any future changes?

Sure, I think that makes sense! 

Also, I'll merge this once the updated time series page is merged so the example links make sense.

**Review by chri2547 — APPROVED** [review]

**Pull request merged**